<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HoloWriter (by NavY LiK), ver. 0.5b</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ICO</normaloff>icon.ICO</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="X">
         <property name="toolTip">
          <string>Start coordinates in % of screen size</string>
         </property>
         <property name="statusTip">
          <string>Start coordinates in % of screen size</string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>5.000000000000000</double>
         </property>
         <property name="value">
          <double>25.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Y">
         <property name="toolTip">
          <string>Start coordinates in % of screen size</string>
         </property>
         <property name="statusTip">
          <string>Start coordinates in % of screen size</string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>5.000000000000000</double>
         </property>
         <property name="value">
          <double>80.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Font:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="font">
         <property name="toolTip">
          <string>Font size</string>
         </property>
         <property name="statusTip">
          <string>Font size</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="colorButton">
         <property name="toolTip">
          <string>Color of output text</string>
         </property>
         <property name="statusTip">
          <string>Color of output text</string>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Text lines:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Double-click a line to replace text. &lt;/p&gt;&lt;p&gt;You also may use comma sign! It would be replaced to false-comma :)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="statusTip">
          <string>Double-click a line to replace text.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="plus">
         <property name="toolTip">
          <string>Add a new line</string>
         </property>
         <property name="statusTip">
          <string>Add a new line</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="minus">
         <property name="toolTip">
          <string>Remove selected line</string>
         </property>
         <property name="statusTip">
          <string>Remove selected line</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="multiLine">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Feature's not availible in BETA!</string>
      </property>
      <property name="statusTip">
       <string>Feature's not availible in BETA!</string>
      </property>
      <property name="text">
       <string>Multi-Line text?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Time settings</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Start delay:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="preDelay">
         <property name="toolTip">
          <string>How long cursor blinks before text starts to appear</string>
         </property>
         <property name="statusTip">
          <string>How long cursor blinks before text starts to appear</string>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Delay between lines:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="lineTime">
         <property name="toolTip">
          <string>How long each line would stay on a screen after completion</string>
         </property>
         <property name="statusTip">
          <string>How long each line would stay on a screen after completion</string>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>1000</number>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Delay between letters:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="letterTime">
         <property name="toolTip">
          <string>Time between letters appearing on a screen</string>
         </property>
         <property name="statusTip">
          <string>Time between letters appearing on a screen</string>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Sounds</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Start sound:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="startSound">
         <property name="text">
          <string>audiobank\scifi\scenery\doors\lock.wav</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Letter sound:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="letterSound">
         <property name="text">
          <string>audiobank\nomadmod\switches\click.wav</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>End sound:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="endSound">
         <property name="text">
          <string>audiobank\scifi\scenery\doors\lock.wav</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="activateByTrigger">
      <property name="toolTip">
       <string>Would this trigger be activated by player in zone? If not, only activated by scripts</string>
      </property>
      <property name="statusTip">
       <string>Would this trigger be activated by player in zone? If not, only activated by scripts</string>
      </property>
      <property name="text">
       <string>Activate by triggerzone enter</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="makeAScript">
      <property name="toolTip">
       <string>Makes an FPI script</string>
      </property>
      <property name="statusTip">
       <string>Makes an FPI script</string>
      </property>
      <property name="text">
       <string>Make a script!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
